### ISSUE TITLE
Unify Props Location schemas

### ISSUE THREAD

**Issue by subdavis** [girder/girder_web_components#55]
The decision to pass `id` and `type` around locally for some components I made when uploader uses `_id` and `_modelType` was a bad decision.

Can we use either 1 or the other?  I'm leaning toward just using the girder schema key names with underscores since consumer applications may make calls to girder that also result in objects of the same schema that they could then pass into GWC components.

Trying to use uploader and databrowser together in demo_site is already annoying because of the differences in their props.

CC @zachmullen @matthewma7

**Comment by zachmullen** [comment]
Sure, the choice to use `_id` and `_modelType` was made because those are the property names returned from the REST API.

**Comment by subdavis** [comment]
Yeah, my original motivation was to isolate GWC from future changes in what is returned from the REST API by having a single conversion function per component, but that seems to have caused me more trouble than it was worth. 

I'd rather fix it now before anything relies on this library.
